pkg/hoarder/cid_source.go

```diff
@@ -67,9 +71,31 @@ func (bitswap_cid_source *BitswapCIDSource) Type() string {
 	return "bitswap"
 }
 
-func read_content_from_file() ([]byte, cid.Cid, error) {
+//Reads CID and content from a given file. Starting idea for the file format should be CID CONTENT([]byte array)\n.
+func read_content_from_file(conf *config.Config) ([]byte, cid.Cid, error) {
```

Comment:
I think you could leave the content aside, it's probably a random one. Furthermore, it isn't relevant for the Cid-Hoarder to know the exact bytes of a given CID.
It would also simplify the step for exporting/importing the CIDs

Reply:
If I leave out the content I would need to change the interface type:
```go
type CidSource interface {
	GetNewCid() ([]byte, cid.Cid, error)
	Type() string
}
```
Every time a GetNewCid() is called it returns the CID , it's content and an nil error if all went well.

Reply:
Yep, although it is never used. 
You can return an empty `[]byte` and it won't crash, it will also save you some time parsing the file with the CIDs.
Might happen that you want to track a CID from where you don't know the content yet

Reply:
Okay makes sense. What do you think would be best to change the interface (which is used I believe in the ``NewRandomCidGen``) or just return a nil type in the place of ``[]byte`` when reading from a file?
The usage is here: 
```go
// TODO: is it worth keeping the content?
// getRandomContent returns generates an array of random bytes with the given size and the composed CID of the content
func genRandomContent(byteLen int) ([]byte, cid.Cid, error) {
	// generate random bytes
	content := make([]byte, byteLen)
	rand.Read(content)

	//TODO do we have to have different CID types?
	// configure the type of CID that we want
	pref := cid.Prefix{
		Version:  1,
		Codec:    cid.Raw,
		MhType:   mh.SHA2_256,
		MhLength: -1,
	}

	// get the CID of the content we just generated
	contID, err := pref.Sum(content)
	if err != nil {
		return content, cid.Cid{}, errors.Wrap(err, "composing CID")
	}

	log.Infof("generated new CID %s", contID.Hash().B58String())
	return content, contID, nil
}
```
Yeah I saw that //TODO now.

Reply:
> What do you think would be best to change the interface or just return a nil type in the place of []byte when reading from a file?
mmmm, I would just return an empty `[]byte` for now, it might be useful in the future